.changeset/gentle-needles-own.md

```diff
@@ -5,6 +5,5 @@
 "section": other
 ---
 
-Adds `getLocalEdits` method to OrderedEditSet interface and removed `./test/EditLog` export
-
+Added getLocalEdits method to OrderedEditSet interface and removed unnecessary test export
```

Comment:
For my understanding, why do we not like `\``s?

Reply:
That's fair. I guess I'm biased a bit more in the other direction (I prefer clear delineation between English and code concepts. But creating a consistent reader experience is the primary objective, so I'm happy with whatever guidance we have, so long as I know what it is 🙂